### PR TITLE
add PID for ECCN M0

### PR DESCRIPTION
**Pull request by bradanlane**
The Bradán Lane STUDIO has created a small PCB using the SAMD21G18 with 4MB of extra flash memory (intended for application data or for CircuitPython code and libraries). It also contains touch pads, a buzzer, and LEDs.

The design is suitable for created a macropad or a small game.

This pull request is to provide a PID for the ECCN M0 for both C/C++ programming and CircuitPython.

### CODE REVIEW

File: 1209/5687/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: ECCN M0
+owner: Bradán.Lane.STUDIO
+license: MIT
+site: https://bradanlane.com/
+source: https://gitlab.com/bradanlane/eccn_m0
+---
+The ECCN M0 is a small SAMD21G18 design with 4MB flash, touch pads, LEDs, and a buzzer. It supports both C/C++ and CircuitPython. An example use for teh the ECCN M0 would be a 3-button macropad.
```
Comment (tannewt) [inline 1209/5687/index.md:9]:
```suggestion
The ECCN M0 is a small SAMD21G18 design with 4MB flash, touch pads, LEDs, and a buzzer. It supports both C/C++ and CircuitPython. An example use for the the ECCN M0 would be a 3-button macropad.
```

**Review by tannewt — CHANGES_REQUESTED** [review]
Please link to the software that this will run and use the PID. (For example, your branch of CircuitPython that adds support.)

Hardware files are Kicad and fine.

**Comment by bradanlane** [comment]
The

How do I include a "source" url for the hardware design plus a "source" url for the CircuitPython firmware as there is only a single field?

The branch for CircuitPython is:

https://github.com/bradanlane/circuitpython/tree/bls_coin_m0

A PR merge is pending review.

**Comment by bradanlane** [comment]
_sorry for the typos in the code change commit comment_

File: 1209/5687/index.md (hunk 1 above)
Comment (bradanlane) [reply 1209/5687/index.md]: corrected typo

**Pull request merged**